satpy/readers/fci_l1c_nc.py

```diff
@@ -29,8 +29,9 @@
 For the Product User Guide (PUG) of the FCI L1c data, see `PUG`_.
 
 .. note::
+    This reader supports datas from both IDPF-I and IQT-I processing facilities.
```

Comment:
```suggestion
    This reader support datas from both IDPF-I and IQT-I processing facilities.
```

Reply:
Did you maybe mean this?
```suggestion
    This reader supports data from both IDPF-I and IQT-I processing facilities.
```

Reply:
yes, obviously, sorry...

Reply:
ok, thanks for spotting! I committed the second.